README.md

```diff
@@ -31,7 +31,7 @@ For other platforms see [hjson.github.io](https://hjson.github.io).
 
 # Compiling
 
-The easiest way to use hjson-cpp is to simply include all of the files from the folders `src` and `include` into your own project. The only requirement is that your compiler fully supports C++11.
```

Comment:
The files in `src` are needed, unless you compile a library or use CMake as described further down in the readme.